### ISSUE TITLE
problems importing postman collection

### ISSUE THREAD

**Issue by gerardgillan** [sttrayno/CML-Personal-Guide#3]
Hi All,

I am having difficulty importing the Postman collection (VIRL-CML 2.0 API.postman_collection.json) into postman, and receive an "Error while importing: format not recognized" message.

Not sure if it's just something I might be doing wrong or if anyone else has experienced this and if so was able to get it to work.

**Comment by gerardgillan** [comment]
Hi Stuart,

Thanks for getting back to me. I am currently running Postman Version 8.2.3 on RHEL7 and have also tried it with Version 8.3.1 on Windows 10.

Regards,

Gerard



Version 8.2.3

**Comment by gerardgillan** [comment]
Hi Stuart,

I've managed to successfully import the Postman collection (VIRL-CML 2.0 API.postman_collection.json). I copied the contents of the file from github and saved it locally with the VIRL................ .json name. This then imported successfully. Not sure what I had done but it's working now. 

I've found your post on using the CML2 API really helpful and informative, thank you very much.

Regards,

Gerard
